src/main/java/core/gui/theme/ImageUtilities.java

```diff
@@ -352,369 +352,318 @@ public static ImageIcon getWideImageIcon4Veraenderung(int value, boolean current
-				if (svgURI != null) {
-					icon.setSvgUniverse(new SVGUniverse());
-					icon.setPreferredSize(new Dimension(width, height));
-					icon.setAutosize(SVGIcon.AUTOSIZE_BESTFIT);
-					icon.setInterpolation(INTERP_BILINEAR);
-					icon.setAntiAlias(true);
-					icon.setSvgURI(svgURI);
-					return icon;
-				}
-			}
-		} catch (Exception e) {	}
-
-		// To avoid infinite loop but will never happen
-		if (!key.equals(HOClassicSchema.EMPTY_SVG))
-			return 	getSvgIcon(HOClassicSchema.EMPTY_SVG);
-		else
-			return icon; // generate an exception, will never happen
-	}
-
-	public static double getBrightness(Color colour) {
-		return Math.sqrt(0.241 * colour.getRed()*colour.getRed()
-				+ 0.691 * colour.getGreen()*colour.getGreen()
-				+ 0.068 * colour.getBlue()*colour.getBlue());
-	}
-
-	private static String getHexColor(Color colour) {
-		return "#" + String.format("%1$02X", colour.getRed()) +
-				String.format("%1$02X", colour.getGreen()) +
-				String.format("%1$02X", colour.getBlue());
-	}
-
-	public static Color getColorFromHex(String hexColour) {
-		return new Color(Integer.valueOf(hexColour.substring(1, 3), 16),
-				Integer.valueOf(hexColour.substring(3, 5), 16),
-				Integer.valueOf(hexColour.substring(5, 7), 16));
-	}
+        return getJerseyIcon(posid, taktik, trickotnummer, 20);
+    }
+
+    public static Icon getJerseyIcon(int posid, byte taktik, int trickotnummer, int size) {
+        String key = "trickot_" + posid + "_" + taktik + "_" + trickotnummer + "_" + size;
+        Icon komplettIcon = ThemeManager.getIcon(key);
+
+        if (komplettIcon == null) {
+            Color jerseyColor = getJerseyColorByPosition(posid);
+
+            double brightness = ImageUtilities.getBrightness(jerseyColor);
+            Color textColor = brightness < 130 ? Color.WHITE : Color.BLACK;
+
+            Map<Object, Object> colorMap = Map.of("jerseyColor", jerseyColor,
+                                                  "collarColor", jerseyColor,
+                                                  "outlineColor", textColor);
+            int width = size;
+            int height = Math.round(size * 16f / 20f);
+            Icon jerseyIcon = IconLoader.get().loadSVGIcon("gui/bilder/jerseys.svg",
+                                                        width, height, true, colorMap);
+
+            Icon numberIcon = EmptyIcon.create(0);
+            if (trickotnummer > 0 && trickotnummer < 50) {
+                int baseline = Math.round(height * 13f / 16f);
+                int fontSize = Math.round(height * 8f / 16f);
+                numberIcon = new TextIcon(String.valueOf(trickotnummer),
+                                          textColor,
+                                          new Font(Font.SANS_SERIF, Font.BOLD, fontSize),
+                                          width, height, baseline);
+            }
+            komplettIcon = new OverlayIcon(jerseyIcon, numberIcon, size, size);
```

Comment:
The formatting changes make it hard to see what the actual change, but this is the bulk of it right here, right?

Reply:
Yes. I'll try to fix the commit so it doesn't change the line endings.

Reply:
I have recommitted the changes without the changes to the formatting.